index.js

```diff
@@ -52,10 +52,10 @@ program
 
 program
   .command('serve [root]')
-  .description('build then serve a website from a directory')
+  .description('build then serve a website from a directory, and open a live preview of the website')
   .option('-f, --force-reload', 'force a full reload of all site files when a file is changed')
-  .option('-p, --port <port>', 'port for server to listen on (Default is 8080)')
-  .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .option('-p <port>, --port <port>', 'port for server to listen on (Default is 8080)')
+  .option('-s <file>, --site-config <file>', 'specify the site config file (default: site.json)')
```

Comment:
This change breaks the options: `-p` and `-s` do not have an effect, and the long options `--port` and `--site-config` are not recognised.

The reason is because the library we are using for the CLI, [commander.js](https://github.com/tj/commander.js), parses the first argument to `.option` [as a list of flags](http://tj.github.io/commander.js/#Command.prototype.option):

> The `flags` string should contain both the short and long flags, separated by comma, a pipe or space.

I don't think we can make this specific change without breaking the CLI.

Reply:
Hi @Xenonym, thanks for the explanation. I should have tested it myself. I got it now. Then I will remove my modification.